### PR TITLE
Feature/acceptance and Mock readme

### PR DESCRIPTION
**Pull request by jcalderin**

### CODE REVIEW

File: test/acceptance/Readme.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+### Prerequisites:
+
+- Python 2.7
+- pip installed (http://docs.python-guide.org/en/latest/starting/install/linux/)
+- virtualenv installed (pip install virtualenv) (optional).
+Note: We recommend the use of virtualenv, because is an isolated working copy of Python which allows you to work on a specific project without worry of affecting other projects.
+
+##### Environment preparation:
+
+- If you are going to use a virtual environment (optional):
```
Comment (fgalan) [inline test/acceptance/Readme.md]: Probably the exact command to be created as hint for people not so familiar with virtualenv (as me :)

File: test/acceptance/Readme.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+```
+
+### Tests Coverage (features):
+
+- Git ISSUE-713 - CPr-propagate Fiware-Service header in forwarded requests
+- Git ISSUE-714 - CPr-propagate Fiware-Servicepath header in forwarded requests
+- Git ISSUE-719 - ServicePath for registrations (NGSI9) (not recursive)
+- Git ISSUE-322 - Service support for registries (NGSI9)
+- Git ISSUE-715 - CPr-propagate X-Auth-Token header in forwarded requests
+- Git ISSUE-716 - ContextProvicer forwarding for attributes not existing in existing entities
```
Comment (fgalan) [inline test/acceptance/Readme.md]: I guess this list matches the current set of tests. W

File: test/acceptance/Readme.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+### properties.json
```
Comment (fgalan) [inline test/acceptance/Readme.md]: Should the parameteres in this file be described in the documentation? Or it is self-content? (In the second case, I'd mention that in the documentation, e.g. "See the content of that file about the different parameters to configure, the file is self-descriptibe" or something like that.

File: test/acceptance/Readme.md (hunk 1 above)
Comment (jcalderin) [inline test/acceptance/Readme.md]: Is in the "Environment preparation" section, how to create and how to activate it :)

File: test/acceptance/Readme.md (hunk 3 above)
Comment (jcalderin) [inline test/acceptance/Readme.md]: Parameter configuration described in the Readme.md file -- 60e8ccc

File: test/acceptance/Readme.md (hunk 1 above)
Comment (fgalan) [inline test/acceptance/Readme.md]: Ah, ok, I didn't realize about it.

Maybe is better to use the "code style" for things that are actually literal commands, I mean:

```
Create a virtual environment somewhere, e.g. in ~/venv (`virtualenv ~/venv`)
Activate the virtual environment (`source ~/venv/bin/activate`) (optional)
```

Comment (jcalderin) [inline test/acceptance/Readme.md]: Fixed in a92cfb5

File: test/acceptance/Readme.md (hunk 2 above)
Comment (jcalderin) [inline test/acceptance/Readme.md]: I have done an script to extract the folders of the tests to try to not forget update the list.

NTC

**Comment by fgalan** [comment]
LGTM

**Comment by fgalan** [comment]
Passing the ball to @iariasleon for final LGTM + merge + delete branch

**Comment by iariasleon** [comment]
LGTM

**Pull request merged**